### DISCUSSION TITLE
如何配置VPN？

### DISCUSSION THREAD

**Discussion by lxnan888** [Yuukiy/JavSP#471] [问答 | Q&A] [answered]
### 功能建议

希望大佬做个Linux系统的，跟win一样运行了直接输入要刮削的路径就行了，docker版本的我不知道在哪里改配置文件，不需要ui，终端运行就可以了

### 提交须知

- [X] 我确认使用的是最新版本并且已经阅读过Wiki帮助文档

**Comment by glyh** [comment]
本身就可以在linux下用，请自己构建一下

**Comment by lxnan888** [comment]
>本身可以在linux下使用，请自己构建一下谢谢我已经安装并且可以正常运行了，请问代理地址怎么设置，我这样设置运行会报错
![Uploading IMG_0259.jpeg…]()

**Comment by glyh** [comment]
你图片没上传成功，我这里看不到

**Comment by lxnan888** [comment]
>你的图片没有上传成功，我这里看不到
我找了一个图床，但是貌似要关闭VPN才能访问，麻烦大佬帮忙看看，谢谢拉网址https://s21.ax1x.com/2024/12/13/pAb4wvR.jpg

**Comment by glyh** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/e8fa0957-3cc0-49f1-b736-a158934cb12d)

**Comment by glyh** [comment]
一直有机器人发广告，锁了
